### ISSUE TITLE
Error trying to build user workflow tasks list for user 1  

### ISSUE THREAD

**Issue by lhall690** [nathanwoulfe/Plumber#123]
**Describe the bug**
After putting some content into workflow its not appearing in the approver's 'tasks requiring my approval' pane. It does appear in the 'workflow administration pane',  and the approver can go in and approve the request, so it seems like that section is working, it's just not displaying correctly.

If I look in the workflow logs I can see the error 
'Error trying to build user workflow tasks list for user 1', which I assume is the correlating issue as the timestamp matches when I have opened the pane. The detailed log is 
Jan 07 2020, 4:25PM:  [21] ERROR Workflow.Api.TasksController - Error trying to build user workflow tasks list for user 1  
System.NullReferenceException: Object reference not set to an instance of an object. 
   at Workflow.Api.TasksController.<>c__DisplayClass12_0.<GetFlowsForUser>b__1(WorkflowTaskPoco x) 
   at System.Linq.Enumerable.WhereListIterator`1.MoveNext() 
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection) 
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source) 
   at Workflow.Api.TasksController.<GetFlowsForUser>d__12.MoveNext() 

**Expected behavior**
Tasks should show in 'tasks requiring my approval' pane

**Screenshots**
![image](https://user-images.githubusercontent.com/59559485/71910763-6e987100-316a-11ea-9c76-8f8e0997794f.png)
![image](https://user-images.githubusercontent.com/59559485/71910933-b9b28400-316a-11ea-9a99-50b422db7f69.png)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser Chrome
 - Version 79.0.3945.88 

**Plumber version**
Which version of Plumber does the bug occur in? 1.1.11
Umbraco 7.15.3

Thank you :)

**Comment by lhall690** [comment]
@nathanwoulfe 
Hi Nathan, I rolled back to v1.10.1 and this issue has resolved.
Cheers

**Comment by nathanwoulfe** [comment]
Interesting... Thanks for letting me know, will look into what's happening...

**Comment by nathanwoulfe** [comment]
@lhall690 I can't reproduce this one - have update my local site to 7.15.3, running latest codebase, and no error. Is there anything else in the logs that might be useful?

**Comment by lhall690** [comment]
Hi Nathan - unfortunately, I have deleted the Umbraco instance where I had 1.11.1 running so dont have access to the logs apart from what I screenshotted above. But I dont think there was anything else in there that would have been useful, unless there was a problem with my specific instance - however it was just a demo umbraco site, and the new instance with 1.10.1 is the same demo site so I dont think it would be that.

Anyway I'm happy to close this ticket down as I have resolved the issue by rolling back.
